apps/webapp/src/components/header/header.tsx

```diff
@@ -1,17 +1,28 @@
-import { Navbar } from './navbar';
+import { LayoutLoaderResult } from '../layout.tsx';
 import { Link, useLoaderData } from 'react-router-dom';
-import { PagePath } from '../metadata/paths.ts';
+import { MessageWarningIcon } from '../../icons/message-warning.tsx';
+import { MobileNavMenu } from './mobile-nav-menu.tsx';
+import { Navbar } from './navbar';
+import {
+  NetworksPopover,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@penumbra-zone/ui';
 import Notifications from './notifications.tsx';
+import { PagePath } from '../metadata/paths.ts';
 import { TabletNavMenu } from './tablet-nav-menu.tsx';
-import { LayoutLoaderResult } from '../layout.tsx';
-import { NetworksPopover } from '@penumbra-zone/ui';
-import { MobileNavMenu } from './mobile-nav-menu.tsx';
+
+// Infinite-expiry invite link, provided by Henry, to the #web-ext-feedback
```

Comment:
Think we can axe the `provided by Henry` part

Reply:
My thinking was that we might want to know whose account this link is tied to in the future, right? If not though, I'll remove that.

Reply:
Ah. I'm not sure 🤷‍♂️. Feel free to add that context to the code comment and it should be good to keep the reference to Henry.

Reply:
OK will do.